ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```
@@ -846,6 +846,18 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-winc
+- as: aws-ipi-ovn-efs-hypershift-mgmt-longduration-f7-destructive
+  cron: 13 18 4,18 * *
+  steps:
+    cluster_profile: aws-qe
```

```suggestion
    cluster_profile: aws-qe
    dependency_overrides:
      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.19
```
The **[error](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/63740/pull-ci-openshift-release-master-ci-operator-config/1911684992115675136)** already shows you missed dependencies
```console
time="2025-04-14T07:40:07Z" level=info msg="Configs reloaded" duration=2.766485189s
time="[2](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/63740/pull-ci-openshift-release-master-ci-operator-config/1911684992115675136#1:build-log.txt%3A2)025-04-14T07:40:09Z" level=error error="failed to validate configuration openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml: invalid configuration: tests[57].literal_steps.pre[37].dependencies[0]: cannot determine source for dependency \"ci-index\" - this dependency requires an operator bundle configuration, which is not configured"
time="2025-04-14T07:40:09Z" level=fatal msg="error validating configuration files"
```